integration_test/src/test_runner/mod.rs

```diff
@@ -654,7 +679,7 @@ fn get_column_count(stmt: &Statement<Allocated, NoResult, AutocommitOn>) -> Resu
     }
 }
 
-fn fetch_row(stmt: &Statement<Allocated, NoResult, AutocommitOn>) -> Result<bool, Error> {
+fn fetch_row(stmt: &Statement<Allocated, NoResult, AutocommitOn>) -> Result<bool> {
```

Comment:
I'm curious, was there a change made that required the `Error` to be added to the result?

Reply:
I made this refactor to be more consistent with (most of) the rest of this codebase and with the mongosql-rs codebase where we often define in each module an `module::Error` enum and a `module::Result<T>` type defined as `std::Result<T, module::Error>`. Here, I initially made a `test_generator::Error` enum and a `test_generator::Result<T>` but then realized I didn't need that and could just use the `test_runner::Error/Result` types. At that point, I decided to do the refactor!